tuf/api/metadata.py

```diff
@@ -289,8 +289,11 @@ def from_dict(cls, signed_dict: JsonDict) -> 'Signed':
 
         # Create empty object with default or parametrized constructor with
         # default arguments.
-        obj = cls()
-        obj._type = signed_dict['_type']
+        # Warnings about rules E1120 and W0212 are not relevant here because
+        # cls is most likely a child class of "Signed" and we need to setup
+        # the appropriate fields.
+        obj = cls() # pylint: disable=E1120
```

Comment:
🤔 this error is `no-value-for-parameter`, which is that we're creating an instance of the class without passing values `__init__`'s arguments. Seems OK given the pattern we decided to adopt here.